app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt

```diff
@@ -107,7 +106,7 @@ open class FileListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        sortType = UISettings(requireContext()).sortType
+        fileListViewModel.sortType = UISettings(requireContext()).sortType
```

Comment:
Get from viewmodel instead ;)

Reply:
`fileListViewModel.sortType ?: UISettings(requireContext()).sortType`

Reply:
It's done 😁